platform/jewel/ide-laf-bridge/src/main/kotlin/org/jetbrains/jewel/bridge/component/JBPopupRender.kt

```diff
@@ -0,0 +1,181 @@
+package org.jetbrains.jewel.bridge.component
+
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.awt.ComposePanel
+import androidx.compose.ui.input.key.KeyEvent
+import androidx.compose.ui.layout.*
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.platform.LocalWindowInfo
+import androidx.compose.ui.semantics.popup
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.unit.*
+import androidx.compose.ui.util.fastForEach
+import androidx.compose.ui.util.fastMap
+import androidx.compose.ui.util.fastMaxBy
+import androidx.compose.ui.util.fastRoundToInt
+import androidx.compose.ui.window.PopupPositionProvider
+import com.intellij.openapi.ui.popup.JBPopupFactory
+import com.intellij.ui.awt.RelativePoint
+import java.awt.Dimension
+import java.awt.Point
+import java.awt.Rectangle
+import org.jetbrains.jewel.bridge.LocalComponent
+import org.jetbrains.jewel.foundation.modifier.thenIf
+import org.jetbrains.jewel.ui.component.PopupRender
+import org.jetbrains.jewel.ui.component.PopupRender.Arguments
+
+internal object JBPopupRender : PopupRender {
+    @Composable
+    override fun Popup(arguments: Arguments, content: @Composable () -> Unit) {
+        JBPopupLayout(arguments, content)
+    }
+}
+
+@Composable
+private fun JBPopupLayout(arguments: Arguments, content: @Composable () -> Unit) {
```

Comment:
This composable was heavily inspired by the current Popup component. Unfortunatelly, we don't have access to the same methods (as most of them are internal), so I'm using the ComposePannel for it.